src/sailfishapp.cpp

```diff
@@ -40,10 +40,73 @@
 #include <QDir>
 #include <QQmlEngine>
 
+/*!
+  \namespace SailfishApp
+  \inheaderfile sailfishapp.h
+  \ingroup libsailfishapp-index
+  \inmodule libsailfishapp
+  \brief The SailfishApp namespace allows the easy creation of SailfishOS applications.
+
+  When creating a QT application for use on SailfishOS, the functions in the \c
+  SailfishApp namespace provide a simple and minimal way to manage your
+  application lifecycle, provide a QML interface, a QT event loop, and manage
+  resources.
+
+  To use libsailfishapp in your project:
+
+  \list 1
+      \li Add \c {BuildRequires: pkgconfig(sailfishapp)} to your .spec file
+      \li Set \c {TARGET = harbour-yourappname} in your .pro file
+      \li Add \c {CONFIG += sailfishapp} to your .pro file
+      \li Include \c {sailfishapp.h} in your .cpp file
+      \li Use the \c {SailfishApp::} methods in your \c main()
+  \endlist
+
+  Example of usage:
+
+  \code
+  #include <QtQuick>
+  #include <QDebug>
+  #include <sailfishapp.h>
+
+  int main(int argc, char *argv[])
+  {
+      QScopedPointer<QGuiApplication> app(SailfishApp::application(argc, argv));
+      QScopedPointer<QQuickView> view(SailfishApp::createView());
+
+      view->setSource(SailfishApp::pathTo(QString("qml/main.qml")));
+      view->show();
+
+      return app->exec();
+  }
+  \endcode
+*/
 
 namespace SailfishApp
 {
 
+/*!
+  \fn QGuiApplication * SailfishApp::application(int &argc, char **argv)
+
+  \brief Create a new QGuiApplication instance configured for use on SailfishOS
+
+  Creates and configures a QGuiApplication ready to be used as part of a
+  SailfishOS application.
```

Comment:
Should be maybe "Sailfish application", but definitely "Sailfish OS" instead of "SailfishOS". I am not fully sure about the guidelines, so rather check with someone more involved in the UX design :) Applies to all occurrences of "Sailfish".

Reply:
I've gone through and changed references to "Sailfish OS" for the operating system and "Sailfish app" or "Sailfish application" for the application. Hopefully this is correct (I'll check). It is at least now consistent throughout.